tests/Io/MultipartParserTest.php

```diff
@@ -696,14 +696,14 @@ public function testUploadTooLargeFile()
         $data .= "Content-Disposition: form-data; name=\"file\"; filename=\"hello\"\r\n";
         $data .= "Content-type: text/plain\r\n";
         $data .= "\r\n";
-        $data .= "world\r\n";
+        $data .= str_repeat('world', 1024) . "\r\n";
```

Comment:
The test makes perfect sense to me, but I feel that this is a bit misplaced here. Maybe add a separate test case for this? :+1:

Reply:
Done :+1: